src/add-patient-link.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { ConfigurableLink } from '@openmrs/esm-framework';
+import { Add20 } from '@carbon/icons-react';
```

Comment:
Use the explicit module import, should be something like `@carbon/icons-react/es/components/add/20` or so